renovate_presets/charm.json5

```diff
@@ -82,6 +89,19 @@
       "extractVersionTemplate": "^juju-(?<version>.*)$",
       "datasourceTemplate": "github-releases",
       "versioningTemplate": "semver"
+    },
+    // Update microk8s to latest release
+    {
+      "customType": "regex",
+      "fileMatch": ["^(workflow-templates|\\.github/workflows)/[^/]+\\.ya?ml$"],
+      "matchStrings": [
+        "microk8s-snap-channel: (?<currentValue>.*?)-strict/stable +# renovate: latest microk8s"
+      ],
+      "depNameTemplate": "Microk8s",
+      "packageNameTemplate": "canonical/microk8s",
+      "datasourceTemplate": "github-releases",
+      "versioningTemplate": "loose",
```

Comment:
Nope.

Reply:
> really? I would've thought x.y works with renovates semver—what happens?

From renovate logs:
```
DEBUG: Dependency canonical/microk8s has unsupported/unversioned value 1.29 (versioning=semver)
DEBUG: Skipping canonical/microk8s because no currentDigest or pinDigests
```

> is there something stricter we can use instead of loose so that renovate can detect major vs minor updates?

[semver-coerced](https://docs.renovatebot.com/modules/versioning/semver-coerced/) seems to work.